### ISSUE TITLE
A11y_AzureMachineLearningExtension_DataSets_Keyboard: The options that are visible on mouse right click for a resource(dataset) i.e., Preview dataset/Unregister dataset options are not available for keyboard users.

### ISSUE THREAD

**Issue by NiharikaTipirishetty** [microsoft/vscode-tools-for-ai#1260]
**Github Tags:**
#A11yMAS;#A11yTCS;#WCAG2.1.1;#Keyboard;#Win32;#DesktopApp;#A11ySev1;#SH-AzureMachineLearningExtension-Win32-Aug2021;#Development and Test Resources for VS and VSCode AI Tools;

### Environment Details:
Visual Studio Code

### Note:
This issue is observed for all resources i.e., Experiments, Models, Compute, EndPoints etc.

### Repro Steps:

1. Hit the [Azure Machine Learning Studio](https://ml.azure.com/) URL and sign in with appropriate credentials.
2. Tab till Notebooks in the left pane and press enter. Select any file from the pane, tab till "Edit in VS code "option in command bar and press enter.
3. A pop up will appears on the screen to open VS Code app, tab till 'Open' button and press enter
4. Tab till Azure in the left pane and press enter, tab till any subscription ex: ML.Net Model Builder Demo and press enter.
5. Tab till any workspace ex: Basic Workspace under the subscription and press enter.
6. Navigate to Datasets/Models/Experiments etc and right click on the resource.
7. The options that are visible on mouse right click i.e., Preview dataset/Unregister dataset options are not available for keyboard users.

### Actual Result:
The options that are visible on mouse right click for a resource i.e., Preview dataset/Unregister dataset options are not available for keyboard users.

### Expected Result:
The options that are visible on mouse right click for a resource i.e., Preview dataset/Unregister dataset options should be available for keyboard users as well. A shortcut can be provided to expand the menu for keyboard users.

### User Impact:
Keyboard users will not be able to access the options if there is no shortcut provided.

**Comment by tbombach** [comment]
I was able to access the context menu using the shortcut `Shift+F10`.

1. I followed the above steps to navigate to datasets
2. I used the shortcut `Shift+F10` and was able to see the context menu with the Preview and Unregister options
3. I was able to navigate the context menu using up/down arrows and activate an option with Enter

So the extension does provide a way to activate the context menu via keyboard. Perhaps we can make it more discoverable by having the shortcut read out for screen readers

**Comment by tbombach** [comment]
Closing this since users are able to activate the context menu via the keyboard

**Comment by NiharikaTipirishetty** [comment]
issue is not repro, closing the bug
#Closed;

[1260_no issue - Trim.zip](https://github.com/microsoft/vscode-tools-for-ai/files/11015776/1260_no.issue.-.Trim.zip)
